### PR TITLE
Update the installation instructions for Linux

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -155,9 +155,11 @@ Dir.chdir "/usr"
...
 ####################################################################### script
 if RUBY_PLATFORM.to_s.downcase.include?("linux")
-  <<-EOABORT
-  You're on Linux! Use the Linuxbrew installer:
-    #{Tty.underline}http://linuxbrew.sh/#{Tty.reset}"
+  abort <<-EOABORT
+  To install Linuxbrew, see the installation instructions at
+    #{Tty.underline}https://docs.brew.sh/Linuxbrew#{Tty.reset}
+  or paste at a terminal prompt
+    sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"
```
Comment (MikeMcQuaid) [inline install]: I think it's probably just worth recommending the latter.

Comment (sjackman) [reply install]: Done

**Comment by MikeMcQuaid** [comment]
Nice work @sjackman!

**Comment by sjackman** [comment]
Thanks for merging, Mike!

**Pull request merged**